packages/i18n/src/fr.ts

```diff
@@ -0,0 +1,66 @@
+import { Language } from './types';
+
+// prettier-ignore
+const language: Language = {
+  code:             'fr',
+  schema:           (issue) => `Mauvais typage: Attend ${issue.expected} mais reçois ${issue.received}`,
+  specific: {
+    bic:            (issue) => `Mauvais BIC: Reçu ${issue.received}`,
+    bytes:          (issue) => `Mauvais bytes: Attend ${issue.expected} mais reçois ${issue.received}`,
+    creditCard:     (issue) => `Mauvaise carte de crédit: Reçu ${issue.received}`,
+    cuid2:          (issue) => `Mauvais Cuid2: Reçu ${issue.received}`,
+    custom:         (issue) => `Mauvais input: Reçu ${issue.received}`,
+    customAsync:    (issue) => `Mauvais input: Reçu ${issue.received}`,
+    decimal:        (issue) => `Mauvaise décimale : Reçu ${issue.received}`,
+    email:          (issue) => `Mauvais email: Reçu ${issue.received}`,
+    emoji:          (issue) => `Mauvais emoji: Reçu ${issue.received}`,
+    endsWith:       (issue) => `Mauvaise fin: Attend ${issue.expected} mais reçois ${issue.received}`,
+    equal:          (issue) => `Mauvaise valeur: Attend ${issue.expected} mais reçois ${issue.received}`,
+    excludes:       (issue) => `Mauvais contenu: Attend ${issue.expected} mais reçois ${issue.received}`,
+    finite:         (issue) => `Mauvais entier: Reçu ${issue.received}`,
+    hash:           (issue) => `Mauvais hash: Reçu ${issue.received}`,
+    hexColor:       (issue) => `Mauvaise couleur hexa: Reçu ${issue.received}`,
+    hexadecimal:    (issue) => `Mauvais hexadécimal: Reçu ${issue.received}`,
+    imei:           (issue) => `Mauvais IMEI: Reçu ${issue.received}`,
+    includes:       (issue) => `Mauvais contenu: Attend ${issue.expected} mais reçois ${issue.received}`,
+    integer:        (issue) => `Mauvais entier: Reçu ${issue.received}`,
+    ip:             (issue) => `Mauvaise IP: Reçu ${issue.received}`,
+    ipv4:           (issue) => `Mauvaise IPv4: Reçu ${issue.received}`,
+    ipv6:           (issue) => `Mauvaise IPv6: Reçu ${issue.received}`,
+    isoDate:        (issue) => `Mauvaise date: Reçu ${issue.received}`,
+    isoDateTime:    (issue) => `Mauvaise date et heure: Reçu ${issue.received}`,
+    isoTime:        (issue) => `Mauvais temps: Reçu ${issue.received}`,
+    isoTimeSecond:  (issue) => `Mauvais temps (secondes): Reçu ${issue.received}`,
+    isoTimestamp:   (issue) => `Mauvais timestamp: Reçu ${issue.received}`,
+    isoWeek:        (issue) => `Mauvaise semaine: Reçu ${issue.received}`,
+    length:         (issue) => `Mauvaise taille: Attend ${issue.expected} mais reçois ${issue.received}`,
```

Comment:
I think the length of a string would be closer to “longeur”. “Reçois” should be “reçu”